### PR TITLE
Add article VoR v3

### PR DESCRIPTION
**Pull request by thewilkybarkid**
https://github.com/elifesciences/issues/issues/4781

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
Not seen this failure before /cc @giorgiosironi

**Comment by giorgiosironi** [comment]
Cause: https://github.com/elifesciences/builder/pull/503/files

File: test/SmokeTest.php [hunk 1]
```diff
@@ -100,19 +100,21 @@ public function requestProvider() : Traversable
         ];
         foreach ((new Finder())->files()->name('*.json')->in(__DIR__.'/../data/articles') as $file) {
             $path = '/articles/'.$file->getBasename('.json');
+            $poaMinimum = 1;
+            $vorMinimum = 26231 == $file->getBasename('.json') ? 3 : 1;
```
Comment (thewilkybarkid) [inline test/SmokeTest.php]: Not a fan of this, but 🤷‍♂.

**Comment by giorgiosironi** [comment]
Fixed the build with a revert.

File: data/articles/26231.json [hunk 2]
```diff
@@ -4791,10 +4787,9 @@
           },
           {
             "type": "paragraph",
-            "text": "Here is a link to the <a href=\"#SA1\">decision letter<a>."
+            "text": "Here is a link to the <a href=\"#SA1\">decision letter</a>."
```
Comment (thewilkybarkid) [inline data/articles/26231.json]: Wasn't a (visible) problem until the link immediately after it disappeared. 😅

**Pull request merged**